src/ethereum.ts

```diff
@@ -919,6 +920,58 @@ function get_rlp_encoded_preimage(rawTx, txType) {
   }
 }
 
+// ======
+// TEMPORARY BRIDGE
+// We are migrating from all legacy signing paths to a single generic
+// signing route. If users are attempting a legacy transaction request
+// against a Lattice on firmware v0.15.0 and above, we need to conver
```

Comment:
```suggestion
// against a Lattice on firmware v0.15.0 and above, we need to convert
```

Reply:
Force pushed in second commit: [0eeddd9](https://github.com/GridPlus/gridplus-sdk/pull/312/commits/0eeddd9c8b35092a0487e74ea7c24fed6bdb45de)